app/views/cases/offender_sar/_closure_outcomes_form.html.slim

```diff
@@ -10,5 +10,6 @@
   .grid-row
     .column-two-thirds
       .button-holder
+        = hidden_field :offender_sar, value: "Test"
```

Comment:
If the form consists solely of radio buttons as is the case here, submitting the form with nothing selected leaves the params[:offender_sar] completely unset and it blows up on the `.require(:offender_sar)` in the offender sar params method. This hidden field is there to prevent this error and has no other side effects. 

I'm thinking there should be a more canonical way to do this but I've gone blank on what it is - better ideas welcomed!

Reply:
This is too dirty! Looking at `offender_sar/_closure_outcomes_form.html.slim`, is the `form_for` output not correct?

```
= form_for kase, as: :"#{@correspondence_type_key}", url: url do |f|
    ...
```

Where `@correspondence_type_key` should output `offender_sar`